### PR TITLE
redis: update to 7.0.4

### PR DESCRIPTION
**Pull request by TwinkleToes777**
#### Testing the changes
- I tested the changes in this PR: YES

#### Local build testing
- I built this PR locally for my native architecture, amd64-glibc

### CODE REVIEW

File: srcpkgs/redis/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+version=7.0.4
 revision=1
 makedepends="jemalloc-devel libatomic-devel"
 checkdepends="tcl-devel procps-ng"
 short_desc="Advanced key-value store"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://redis.io"
-changelog="https://raw.githubusercontent.com/antirez/redis/5.0/00-RELEASENOTES"
+changelog="https://github.com/redis/redis/releases/tag/7.0.4"
```
Comment (Chocimier) [inline srcpkgs/redis/template:11]: Please keep with <https://raw.githubusercontent.com/redis/redis/${version%.*}/00-RELEASENOTES>, tag link shows only last point relase changes and is a heavy html.

**Comment by Chocimier** [comment]
You can build with tests locally with `xbps-src -Q pkg redis`. Put things needed only for tests  into `checkdepends=`.

**Comment by paper42** [comment]
this PR should contain one commit with an appropriate commit message, read Contributing.md for more details about this

**Comment by TwinkleToes777** [comment]
Will be done, sorry about that, while I read contributing.md, my eyes did not catch that part the first time.
> this PR should contain one commit with an appropriate commit message, read Contributing.md for more details about this

**Comment by Chocimier** [comment]
When I tried to update before, after adding dependencies small fraction of tests was still failing; please push once you get there.

**Comment by TwinkleToes777** [comment]
> When I tried to update before, after adding dependencies small fraction of tests was still failing; please push once you get there.

It looks like redis 7.0.4 has one or some bugged tests that always fail, at least on amd64-glibc, does that make it illegible for update?

**Comment by TwinkleToes777** [comment]
Closing as I do not know how to solve this. It works fine in production for me even if not all tests pass, if someone wants a newer redis version before someone else updates, they can build it

**Pull request closed without merging**